swatch-contracts/src/main/resources/META-INF/openapi.yaml

```
@@ -1120,6 +1120,118 @@ paths:
       security:
         - customer: [ ]
 
+  /api/rhsm-subscriptions/v1/capacity/products/{product_id}/{metric_id}:
```

```suggestion
  /api/swatch-contracts/v1/capacity/products/{product_id}/{metric_id}:
```

Other paths that were recently ported needs fixing as well:
- /api/rhsm-subscriptions/v2/subscriptions/products/{product_id}
- /api/rhsm-subscriptions/v1/subscriptions/products/{product_id}